TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/ObstacleCoursePath.java

```diff
@@ -52,6 +59,9 @@ public void runOpMode() {
 
         if(isStopRequested()) return;
 
+        // first chunk
```

Comment:
If empty lines were added between the trajectory parts, consider adding spaces between the calling of the trajectory parts here.